### DISCUSSION TITLE
Stats not updating

### DISCUSSION THREAD

**Discussion by JackGraymer** [anuraghazra/github-readme-stats#3675] [Q&A] [answered]
### Describe the bug

The stats are not updating on my github.username.md for the profile.

I have my own instance deployed on vercel working. The banner on github does not update, but when used on a different program or site, like a local jupyter notebook or a markdown online editor like stackedit.io it does show the current/actual data.

### Expected behavior

updated data, bugger numbers and python among other languages

### Screenshots / Live demo link

https://github.com/JackGraymer/JackGraymer shows no python, using the md line:
<img height="250" src="https://github-readme-stats-jackgraymer.vercel.app/api/top-langs/?username=jackgraymer&layout=compact&custom_title=Most%20used%20languages&langs_count=10&include_all_commits=true&hide_progress=false&hide_border=false&theme=dark&hide=">

for example on https://stackedit.io/app# will shows python today
![image](https://github.com/anuraghazra/github-readme-stats/assets/89271771/58c549c9-96e1-4408-b52b-8f93f1883300)

**Comment by qwerty541** [comment, marked as answer]
Hey, @JackGraymer!

At first I want to say that this repository does not provide streak stats card, so if you have any questions related to this card you should ask them in corresponding place https://github.com/DenverCoder1/github-readme-streak-stats

I can guess from my side that your problem most likely related to caching. If you inspect html code of your streak card you will find that GitHub reuploads and cache image on their CDN and img tag leads on `camo.githubusercontent.com` instead of your instance. So updating data on streak card wont be immediate after making contribution, it should take a few hours.

**Reply by JackGraymer** [reply]
After a few hours, did update

**Comment by JackGraymer** [comment]
Right, This one works, is the streak that doesn't, thanks

**Comment by sntgchns** [comment]
I leave this solution that may be useful, you can send a request to update the card:

Obtain your link for your badge url, like "https://camo.githubusercontent.com/<LONG_NUMBER_HERE>"
And do a curl -X PURGE  to the url.

For example I leave the one I use to update the one in my readme.md:
```
curl -X PURGE https://camo.githubusercontent.com/149e45353035d561cab277e62a289488fac9bafeb9659fabe5c171e23d47c7b5/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170693f757365726e616d653d736e746763686e7326636f756e745f707269766174653d7472756526637573746f6d5f7469746c653d736e746763686e732532304769744875622532305374617473266c6f63616c653d656e26686964655f72616e6b3d66616c73652673686f773d64697363757373696f6e735f737461727465642c7072735f6d65726765642c7072735f6d65726765645f70657263656e7461676526696e636c7564655f616c6c5f636f6d6d6974733d747275652673686f775f69636f6e733d74727565267468656d653d6769746875625f6461726b
```
